easybuild/easyconfigs/s/SWIG/SWIG-3.0.10-CrayGNU-18.08-python3.eb

```diff
@@ -0,0 +1,32 @@
+# @author: gppezzi, marcelsc
+
+name = 'SWIG'
+version = '3.0.10'
+
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '1.1'
+
+versionsuffix = '-python%s' % (py_maj_ver)
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('cray-python/%s.%s.%s' % (py_maj_ver, py_min_ver, py_rev_ver), EXTERNAL_MODULE),
```

Comment:
Since the cray-python needs to be updated, could you also use the standard style `"cray-python/%s" % pyver` also here?